defaults/docs-defaults.yaml

```diff
@@ -64,6 +93,128 @@ defaults:
     sri:
       fixed: 0.004430000
       mobile: 0.007970000
+  default_percent_mobile:
+    scope3: 23.600000000
+    sri: 10.000000000
+  default_percent_mobile_by_country:
+    scope3:
+      AGO: 59.384000000
```

Comment:
@bokelley Any chance we can make these iso 2 letter country codes? All our other internal country management is in 2 letter country codes. We could convert but it would be better to keep same pattern I think.